docs/operator-docs/granule-workflows.md

```diff
@@ -0,0 +1,95 @@
+---
+id: granule-workflows
+title: Granule Workflows
+hide_title: true
+---
+
+# Granule Workflows
+
+## Ingest Kinesis
```

Comment:
This document is not describing how to ingest via Kinesis, it's describing how to set up for ingest. To ingest, you'd have to create the kinesis rule, then put a message on the kinesis stream. Maybe we should leave that until next PR, since right now we only use kinesis for CNM messages.

Reply:
Hmm. I don't know it's not showing. <img width="1113" alt="Screen Shot 2020-07-17 at 11 58 59 AM" src="https://user-images.githubusercontent.com/40603352/87806517-01eee500-c825-11ea-84ab-78843a99db57.png">

Reply:
@laurenfrederick Okay. I just added a subtitle and lean in to Kinesis Stream for Ingest.

Reply:
And doesn't actually ingest a granule that is not already ingested.

Reply:
@laurenfrederick Just seeing this. Sorry I didn't notice the email reply. What do suggest? To remove this?